android/src/app/organicmaps/location/LocationHelper.java

```diff
@@ -33,20 +24,12 @@
 import app.organicmaps.util.Listeners;
 import app.organicmaps.util.LocationUtils;
 import app.organicmaps.util.NetworkPolicy;
-import app.organicmaps.util.Utils;
 import app.organicmaps.util.log.Logger;
 
-public enum LocationHelper implements Initializable<Context>, AppBackgroundTracker.OnTransitionListener, BaseLocationProvider.Listener
+public enum LocationHelper implements Initializable<Context>, BaseLocationProvider.Listener
```

Comment:
Why is it not removed completely?

Reply:
Why the LocationHelper is still used? What are the next steps to remove it? Or will it stay?